### PR TITLE
fix(web): Fix for failing to load pictures

### PR DESCRIPTION
**Pull request by lukasdotcom**
My attempt at fixing #14914. I did test that this pull request fixes that bug, and didn't see any weird side effects, but not sure if this is the correct way of doing it.

### CODE REVIEW

**Review by alextran1502 — APPROVED** [review]

File: e2e/test-assets
Comment (danieldietzler) [inline e2e/test-assets:1]: These changes are unrelated. Please revert them.

**Review by danieldietzler — DISMISSED** [review]
